host/FrameBuffer.cpp

```diff
@@ -1277,6 +1279,7 @@ FrameBuffer::Impl::~Impl() {
 
 WorkerProcessingResult FrameBuffer::Impl::sendReadbackWorkerCmd(const Readback& readback) {
     ensureReadbackWorker();
+#if GFXSTREAM_ENABLE_HOST_GLES
```

Comment:
if we move m_readbackWorker into GFXSTREAM_ENABLE_HOST_GLES, we should ideally move related functions like this sendReadbackWorkerCmd as well.